trucks.js

```diff
@@ -501,4 +539,115 @@ var foodTrucks = [
     Facebook: 'https://www.facebook.com/fulltilticecreams',
     Twitter: 'https://twitter.com/fticecream'
   }
-];
\ No newline at end of file
+];
+
+function getTrucks() {
+  return foodTrucks;
+}
+
+function getTruck( day ) {
+
+  var truck;
+
+  function getTruck( obj ) {
```

Comment:
I would avoid using the same name for your inner function as the outer function here. It makes the code more difficult to understand.